python/cuml/dask/common/dask_df_utils.py

```diff
@@ -79,3 +81,29 @@ def to_dask_cudf(futures):
     dfs = [d for d in futures if d.type != type(None)]  # NOQA
     meta = c.submit(get_meta, dfs[0]).result()
     return dd.from_delayed(dfs, meta=meta)
+
+
+def to_dask_df(dask_cudf):
+    """
+    Convert a Dask-cuDF into a Pandas-backed Dask Dataframe.
+
+    :param dask_cudf : dask_cudf.DataFrame
+    :return : dask.DataFrame
+    """
+
+    def to_pandas(df, r):
+        return df.to_pandas()
+
+    c = default_client()
+    delayed_ddf = dask_cudf.to_delayed()
+    gpu_futures = c.compute(delayed_ddf)
+    wait(gpu_futures)
+
+    dfs = [c.submit(
+        to_pandas,
+        f,
+        random.random()) for f in gpu_futures]
```

Comment:
is the random state just to differentiate each task so they don't look like dupes? maybe we should just use indices like `for i, f in enumerate(gpu_futures)`?
Something about the stray random with no seed makes me strangely uncomfortable..

Reply:
This is just to stop Dask's memoization. We do it all over the place.

Reply:
`client.submit` allows us to set the prefix of task names. By default it uses the name of the function and a hash of its contents. I think we can set the prefix to a UUID (which will still be implicitly random), but we will no longer need to explicitly pass it into the functions.